src/main/pages/setup-docker/docker-single-user.adoc

```diff
@@ -312,25 +323,25 @@ docker run -p:9000:9000 \
-|`CHE_IN_VM` |Set to 'true' if this container is running inside of a VM providing Docker such as boot2docker, Docker for Mac, or Docker for Windows. We auto-detect this for most situations, but it’s not always perfect. |auto-detection
+|`CHE_IN_VM` |Set to 'true' if this container is running inside a VM providing Docker such as boot2docker, Docker for Mac, or Docker for Windows. This is auto-detected in most situations, but it is not always perfect. |auto-detection
 |`CHE_LOG_LEVEL` |Logging level of output for Che server. Can be `debug` or `info`. |`info`
-|`CHE_HOST` |IP address/hostname Che server will bind to. Used by browsers to contact workspaces. You must set this IP address if you want to bind the Che server to an external IP address that is not the same as Docker’s. |The IP address set to the Docker host. This does cover 99% of situations, but on rare occassions we are not able to discover this IP address and you must provide it.
-|`CHE_DEBUG_SERVER` |If `true`, then will launch the Che server with JPDA activated so that you a Java debugger can attach to the Che server for debugging plugins, extensions, and core libraries. |`false`
+|`CHE_HOST` |IP address/hostname that the Che server will bind to. Used by browsers to contact workspaces. You must set this IP address if you want to bind the Che server to an external IP address that is not the same as Docker’s. |The IP address set to the Docker host. This covers 99% of situations, but on rare occassions this IP address is not discovered and you must provide it.
+|`CHE_DEBUG_SERVER` |If `true`, it will launch the Che server with JPDA activated so that a Java debugger can attach to the Che server for debugging plugins, extensions, and core libraries. |`false`
 |`CHE_DEBUG_SERVER_PORT` |The port that the JPDA debugger will listen. |`8000`
-|`CHE_DEBUG_SERVER_SUSPEND` |If `true`, then activates `JPDA_SUSPEND` flag for Tomcat running the Che server. Used for advanced internal debugging of extensions. |`false`
-|`CHE_PORT` |The port the Che server will bind itself to within the Che container. |`8080`
+|`CHE_DEBUG_SERVER_SUSPEND` |If `true`, then activates the `JPDA_SUSPEND` flag for Tomcat running the Che server. Used for advanced internal debugging of extensions. |`false`
+|`CHE_PORT` |The port that the Che server will bind itself to within the Che container. |`8080`
 |===
 
-You can find list of envs in https://github.com/eclipse/che/blob/master/dockerfiles/init/manifests/che.env[che.env].
+For a list of environment variables, see https://github.com/eclipse/che/blob/master/dockerfiles/init/manifests/che.env[che.env].
 
-You can create a file with envs you want to pass to che-server:
+You can create a file with the environment variables that you want to pass to the che-server:
```

Comment:
* -> 'pass to che-server' or 'pass to the che-server image'
* mark up the image name